libs/libc/stdlib/lib_atexit.c

```diff
@@ -181,3 +186,70 @@ int atexit(CODE void (*func)(void))
 {
   return atexit_register(ATTYPE_ATEXIT, func, NULL, NULL);
 }
+
+int at_quick_exit(CODE void (*func)(void))
```

Comment:
Do we need to add a separate comment description for `at_quick_exit` or maybe modify existing description of `atexit`?

Reply:
Done.